compute_tools/src/pg_helpers.rs

```diff
@@ -214,7 +214,7 @@ impl Escaping for PgIdent {
         // Find the first suitable tag that is not present in the string.
         // Postgres' max role/DB name length is 63 bytes, so even in the
         // worst case it won't take long.
-        while self.contains(&format!("${tag}$")) || self.contains(&format!("${outer_tag}$")) {
+        while self.contains(&tag.to_string()) || self.contains(&outer_tag.to_string()) {
```

Comment:
Sorry, it seems to be absolutely not principle, for it takes me so long time to understand the problem, 
that after I have understand it, I continue thinking more about the fix:)
There is completely no sense to check for occurrences both of "x" and "xx". If "x" doesn't not present, then obviously "xx" also can not present.

So this check can be simplified just to: `while self.contains(&tag.to_string())`